dftimewolf/metawolf/utils.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Metawolf utilities."""
+
+import json
+import os
+import sys
+import uuid
+
+from pydoc import locate
+from typing import Any, Dict, List, Optional, Union, TYPE_CHECKING
+
+from dftimewolf.lib import errors
+from dftimewolf.cli import dftimewolf_recipes
+from dftimewolf.metawolf import session
+from dftimewolf.lib import resources
+
+if TYPE_CHECKING:
+  from dftimewolf.metawolf import output  # pylint: disable=cyclic-import
+
+DFTIMEWOLF = 'dftimewolf'
+
+LAST_ACTIVE_SESSION = 'last_active_session'
+LAST_ACTIVE_RECIPE = 'last_active_recipe'
+LAST_ACTIVE_PROCESSES = 'last_active_processes'
+SESSION_ID_SETTABLE = 'session_id'
+
+
+class MetawolfUtils:
+  """MetawolfUtils holds a set of utility methods for Metawolf.
+
+  Attributes:
+    recipe_manager (RecipeManager): A DFTimewolf RecipeManager object.
+  """
+
+  def __init__(self, session_path: str = '') -> None:
+    """Initialize Metawolf utilities.
+
+    Args:
+      session_path (str): Optional. The path to the file in which session
+          information should be stored.
+    """
+    self.session_path = session_path
+    self.recipe_manager = self.GetDFTool().RecipeManager()
```

Comment:
I believe you mean `RecipesManager`? (with an extra `s` after `Recipe`) I wonder why tests didn't catch this?

Reply:
Looks like `RecipesManager` is the type, not the method's name. For less confusion, I'm gonna rename the method to  `RecipesManager `